content/assignments/launch-checklist/planet-fetch/_index.md

```diff
@@ -25,11 +25,25 @@ Review the list and decide which planet you want to send our intrepid crew to an
 {{% /notice %}}
 
 In `scriptHelper.js`, you have three functions for this task: `myFetch()`, `pickPlanet()`, and `addDestinationInfo()`.
+
 First, review the comments in `addDestinationInfo()`.
-This is the format of the `innerHTML` for the `missionTarget` div, which you can locate using the `document` parameter of `addDestinationInfo()`.
-`addDestinationInfo()` does not need to return anything.
+This is the format of the `innerHTML` for the `missionTarget` div, which you can locate using the `document` parameter of `addDestinationInfo()`. `addDestinationInfo()` does not need to return anything.
+
 `pickPlanet()` takes in one argument: a list of planets. Using `Math.random()`, return one planet from the list with a randomly-selected index.
-`myFetch()` has some of the code necessary for fetching planetary JSON, however, it is not complete. You need to add the URL and return `response.json()`.
+
+`myFetch()` has some of the code necessary for fetching planetary JSON, however, it is not complete. You need to add the URL to the `fetch` method call and return `response.json()` from the inner function.
+
+
+{{% notice blue "Note" "rocket" %}}
```

Comment:
Formatting is off here. 
I was able to fix this issue by keeping the text at the same level of indentation as the notice block and putting the first sentence immediately after the opening block. You can still have space between sentences in the block, but I think the first one has to be right next to the opening block.

Example:
```
{{% notice blue Note "rocket" %}}
The `replace` method can be used in more powerful ways utilizing regular expressions. We will not cover those here, but you [can read more at MDN.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)
{{% /notice %}}
```